package.json

```diff
@@ -99,7 +99,7 @@
     "jest-enzyme": "^7.0.0",
     "lerna": "^3.0.0",
     "lerna-update-wizard": "^0.11.0",
-    "lint-staged": "^7.0.4",
+    "lint-staged": "^8.1.0",
```

Comment:
can we set `>=` for following peerDependencies in the file https://github.com/telus/tds-community/blob/auto-skeleton-provider/packages/Skeleton/package.json
```
  "peerDependencies": {
    "@tds/core-heading": ">=1.1.5",
    "@tds/core-image": ">=1.0.0",
    "@tds/core-text": ">=1.0.4",
```

Reply:
done.